baseapp/datadog/datadog.go

```diff
@@ -48,6 +48,12 @@ const (
 	DefaultInterval = 10 * time.Second
 )
 
+var (
+	// TimerUnits sets the units used when exporting metrics.Timer metrics. By
+	// default, use the native unit of nanoseconds.
+	TimerUnits = time.Nanosecond
```

Comment:
Two questions about this:

1. Should we break compatibility a set a more useful default, like milliseconds? I'm worried that breaking metric changes are harder to catch than breaking code changes.
2. Should this be a global or a property of the emitter? A global is easier to set and I think you'd only set this once per application, but the global does make the package move in #183 a bit harder if we want to avoid a breaking change.

Reply:
I think accidentally breaking metrics will be really annoying to track down, but its not "technically" and API break?

I think it would be more friendly to have this option be controllable on a per-app basis, defaulting to Nanoseconds.

Reply:
I agree that we should keep the default as nanoseconds, as we have already configured some datadog metric units based on the assumption the timer reports as nanoseconds. As long as this is well documented, I think it's fine to stick with the nanosecond default.

Reply:
Ok, I'll leave this as a global property you can set per application that defaults to nanoseconds. For the package move, I think I'll either make this option only apply in the new package (which will be fine as long as we release this change and the package move together), or I'll make this a `SetTimerUnit` function instead of a variable.